build.sh

```diff
@@ -319,13 +319,53 @@ EOF
   cp libgnark_eip_196.* "$SCRIPTDIR/gnark/build/${OSARCH}/lib"
 }
 
+build_constantine() {
+  echo "#############################"
+  echo "####### build constantine ####"
+  echo "#############################"
+
+  # Skip if OSARCH is linux-gnu-aarch64
+  if [[ "$OSARCH" == "linux-gnu-aarch64" ]]; then
+    echo "Skipping build for OSARCH: ${OSARCH}"
+    return
+  fi
```

Comment:
is this a CI-specific workaround?  it looks like an arm64 linux host env would not be able to build constantine?

Reply:
I skipped Linux arm64 build since nim installer was not compatible with it, and I didn't have an Linux arm environment to test it.
Will test and add in the follow on PR